air-quality-backend/src/etl/forecast/forecast_dao.py

```diff
@@ -42,10 +46,16 @@ def fetch_forecast_data(
     model_base_date: datetime,
 ) -> ForecastData:
     model_base_date_str = model_base_date.strftime("%Y-%m-%d")
-    single_level_data = fetch_cams_data(
-        get_single_level_request_body(model_base_date_str), "single_level.grib"
-    )
-    multi_level_data = fetch_cams_data(
-        get_multi_level_request_body(model_base_date_str), "multi_level.grib"
-    )
-    return ForecastData(single_level_data, multi_level_data)
+    model_base_time = "00"
+    task_params = [
+        (
+            get_single_level_request_body(model_base_date_str, model_base_time),
+            f"single_level_{model_base_date_str}_{model_base_time}.grib",
+        ),
+        (
+            get_multi_level_request_body(model_base_date_str, model_base_time),
```

Comment:
Possibility that creating new files each time this is run could cause issues for user of application?